packages/docusaurus-plugin-content-blog/src/pluginOptionSchema.ts

```diff
@@ -47,9 +47,8 @@ export const PluginOptionSchema = Joi.object({
     .default(DEFAULT_OPTIONS.routeBasePath),
   include: Joi.array().items(Joi.string()).default(DEFAULT_OPTIONS.include),
   exclude: Joi.array().items(Joi.string()).default(DEFAULT_OPTIONS.exclude),
-  postsPerPage: Joi.number()
-    .integer()
-    .min(1)
+  postsPerPage: Joi.alternatives()
+    .try(Joi.equal('ALL').required(), Joi.number().integer().min(1).required())
```

Comment:
No strong opinion on this, was just wondering if `+Infinity` wouldn't make sense?

Reply:
`Math.ceil(20 / Infinity) === 0`, so not sure how it would work, we probably have to handle it differently anyways

Reply:
yes, let's keep it this way then